client/src/header.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class Header extends React.Component {
```

Comment:
Is this component going to manage state, or could it potentially be a functional-stateless component?

Reply:
Good point!  I originally made all my components to manage state just in case I needed it, and didn't get around to refactoring it to a stateless component.